libs/@guardian/source-foundations/src/_deprecated/typography/data.ts

```diff
@@ -3,14 +3,16 @@ import {
 	fontArrayToString,
 	pxStringToNumber,
 	pxToRem,
-} from '../utils/convert-value';
+} from '../../utils/convert-value';
 import type { AvailableFontWeights, ItalicsFontWeights } from './types';
 
 /**
  * Pixel size values for each font that we use in the design system.
  *
  * We assert that the values match the guardian type scale in our
  * {@link [unit test suite](./typography.test.ts)}.
+ *
+ * @deprecated Please typography presets rather than setting individual values
```

Comment:
shall we add a link to the presets docs?

https://guardian.github.io/storybooks/?path=/docs/source-foundations_typography--docs